src/auto/bytes_icon.rs

```diff
@@ -0,0 +1,79 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use Icon;
+use LoadableIcon;
+use ffi;
+use glib;
+use glib::StaticType;
+use glib::Value;
+use glib::object::Downcast;
+use glib::object::IsA;
+use glib::signal::SignalHandlerId;
+use glib::signal::connect;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+use std::boxed::Box as Box_;
+use std::mem;
+use std::mem::transmute;
+use std::ptr;
+
+glib_wrapper! {
+    pub struct BytesIcon(Object<ffi::GBytesIcon>): Icon, LoadableIcon;
+
+    match fn {
+        get_type => || ffi::g_bytes_icon_get_type(),
+    }
+}
+
+impl BytesIcon {
+    #[cfg(any(feature = "v2_38", feature = "dox"))]
+    pub fn new(bytes: &glib::Bytes) -> BytesIcon {
+        unsafe {
+            from_glib_full(ffi::g_bytes_icon_new(bytes.to_glib_none().0))
+        }
+    }
+}
+
+pub trait BytesIconExt {
+    #[cfg(any(feature = "v2_38", feature = "dox"))]
+    fn get_bytes(&self) -> Option<glib::Bytes>;
+
+    fn get_property_bytes(&self) -> Option<glib::Bytes>;
```

Comment:
This property need version 2_38 to be hidden

Reply:
Why? https://developer.gnome.org/gobject/stable/gobject-The-Base-Object-Type.html#g-object-get-property There is no minimum version...

Reply:
It minimum version for class itself and we already have `https://developer.gnome.org/gio/stable/GBytesIcon.html#g-bytes-icon-get-bytes`, so better hide property by setting same version to it.
Maybe this need do gir automatically.

PS. Strange that no set property generated

Reply:
Ah understand why no set, it Write/Construct only.

Reply:
Ok, I add the version then.